frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/ToolJetDbOperations.jsx

```diff
@@ -9,12 +9,13 @@ import { toast } from 'react-hot-toast';
 import Select from '@/_ui/Select';
 import { queryManagerSelectComponentStyle } from '@/_ui/Select/styles';
 import { useMounted } from '@/_hooks/use-mount';
+import { useCurrentState } from '@/_stores/currentStateStore';
 
-const ToolJetDbOperations = ({ currentState, optionchanged, options, darkMode }) => {
+const ToolJetDbOperations = ({ optionchanged, options, darkMode }) => {
   const computeSelectStyles = (darkMode, width) => {
     return queryManagerSelectComponentStyle(darkMode, width);
   };
-
+  const currentState = useCurrentState();
```

Comment:
`currentState`` is used here only to pass down to child components ListRows, CreateRow, UpdateRows and DeleteRows where we have already removed the dependancy. So we can remove currentState from this component as well.